wallet/node/Params.hs

```diff
@@ -0,0 +1,71 @@
+-- | Getter params from Args
+
+module Params
+       ( getNodeParams
+       ) where
+
+import           Universum
+
+import           Mockable              (Fork, Mockable)
+import           System.Wlog           (WithLogger)
+
+import           Pos.Client.CLI        (CommonNodeArgs (..))
+import qualified Pos.Client.CLI        as CLI
+import           Pos.Constants         (isDevelopment)
+import           Pos.Core.Types        (Timestamp (..))
+import           Pos.Genesis           (GenesisContext (..), devAddrDistr, devStakesDistr,
+                                        genesisContextProduction, genesisUtxo)
+import           Pos.Launcher          (NodeParams (..))
+import           Pos.Network.CLI       (intNetworkConfigOpts)
+import           Pos.Security          (SecurityParams (..))
+import           Pos.Update.Params     (UpdateParams (..))
+import           Pos.Util.UserSecret   (peekUserSecret)
+
+getNodeParams ::
+       (MonadIO m, MonadFail m, MonadThrow m, WithLogger m, Mockable Fork m)
+    => CommonNodeArgs
+    -> Timestamp
+    -> m NodeParams
+getNodeParams args@CommonNodeArgs{..} systemStart = do
```

Comment:
There is `getNodeParams` in `Pos.Client.CLI.Params` which looks very similar. Is it different?
Perhaps this file exists by mistake, because `npSecurityParams` shouldn't be in wallet.